README.md

```diff
@@ -182,6 +185,40 @@ npx asari pulls open <github-url>
 # Set the state of an existing pull request to `open`.
 ```
 
+### Working with GitHub Topics
+
+```bash
+npx asari topics list <github-url>
+
+# List all topics.
+```
+
+```bash
+npx asari topics add <github-url> --topic new-app
```

Comment:
Could trim this to:

```suggestion
npx asari topics <add|remove> <github-url> --topic new-app
```

Reply:
Also for moving to `repos` subcommand, we would have `repos add-topic` and `repos remove-topic`